rust/foxglove/src/mcap_writer/mcap_sink.rs

```diff
@@ -65,7 +70,8 @@ impl<W: Write + Seek> WriterState<W> {
                 mcap_channel_id
             }
         };
-        let sequence = self.next_sequence(channel_id);
+
+        let sequence = self.next_sequence(mcap_channel_id);
```

Comment:
This is a bugfix which accounts for the "canonical channel" handling of the rust mcap writer.